examples/src/main/java/io/dapr/examples/workflows/DemoWorkflowClient.java

```diff
@@ -59,6 +59,19 @@ public static void main(String[] args) throws InterruptedException {
       System.out.println("**SendExternalMessage**");
       client.raiseEvent(instanceId, "TestEvent", "TestEventPayload");
 
+      System.out.println(separatorStr);
+      System.out.println("** Registering parallel Events to be captured by allOf(t1,t2,t3**");

```

Comment:
Missing closing brackets - System.out.println("** Registering parallel Events to be captured by allOf(t1,t2,t3) **");